### ISSUE TITLE
Overriden style

### ISSUE THREAD

**Issue by Shikanime** [sfeir-open-source/xlayers#46]
**Describe the bug**
Some object background doesn't have correct color.

![xlayers sketch viewer and codegen 2018-11-07 17-16-23](https://user-images.githubusercontent.com/22115108/48144290-e641e600-e2b0-11e8-9acd-b373255f5fbe.png)

**Expected behavior**
Should have the background color expected by the sketch style specification.

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Browser Google Chrome
 - Version  70.0.3538.77 (Official Build) (64-bit)

**Comment by Shikanime** [comment]
The issue seems to be because of root property override in style parser at:
```TypeScript
private setStyle(obj: any, root: any, style: { [key: string]: string }) {
    root.css = root.css || {};
    obj.css = obj.css || {};
    for (const property in style) {
      if (style.hasOwnProperty(property)) {
        root.css[property] = style[property];
        obj.css[property] = style[property];
      }
    }
  }
```
![devtools - localhost 4200 2018-11-07 17-31-22](https://user-images.githubusercontent.com/22115108/48145378-0377b400-e2b3-11e8-96cf-2d5044c5ab48.png)

**Comment by Shikanime** [comment]
The problem is more important and affects all styles.
For the moment, the problem is solved by xlayers/xlayers/pull/68 and the failure of the text style parser (#22) is temporarily fixed by disabling the parser in question.

**Comment by manekinekko** [comment]
fixed  https://github.com/xlayers/xlayers/pull/68
